### PR TITLE
Draft SEO / OG - mapping multiple fields

### PR DESCRIPTION
**Pull request by boton**
Draft for https://issues.liferay.com/browse/LPS-135478

### CODE REVIEW

File: docs/dxp/latest/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md [hunk 1]
```diff
@@ -17,15 +17,35 @@ Search engine optimization (SEO) refers to the methods used to improve your Page
...
+
+The recommended length for an *HTML title* is under 60 characters.
+
+Default value in HTML title:
+![image](https://user-images.githubusercontent.com/67901/137364376-07ac46f4-fc32-466c-a400-5c498ed6424d.png)
+
+Custom title in HTML title:
+![image](https://user-images.githubusercontent.com/67901/137364659-2f22e395-5a6f-46f1-a847-e424706852b3.png)
+
+For example: `${title} - ${authorName}` produces something like.
```
Comment (austinchiang) [inline docs/dxp/latest/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md]: For example, `${title} - ${authorName}` produces something like:

File: docs/dxp/latest/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md [hunk 2]
```diff
@@ -17,15 +17,35 @@ Search engine optimization (SEO) refers to the methods used to improve your Page
...
 
-The *HTML Title* field defines a Display Page's `<title>` tag. This title is used by search engines to rank your Page and serves as the Page's heading in search engine results. By default, Display Page templates map this field to *Title*, but you can select any other Text field. The recommended length for an HTML title is under 60 characters.
+The *HTML Title* field defines a Display Page's `<title>` tag. This title is used by search engines to rank your Page and serves as the Page's heading in search engine results. By default, Display Page templates map this field to `${title}`, but you can build your title with the mapped fields  that the selector gives you. 
+
+The recommended length for an *HTML title* is under 60 characters.
+
+Default value in HTML title:
+![image](https://user-images.githubusercontent.com/67901/137364376-07ac46f4-fc32-466c-a400-5c498ed6424d.png)
+
+Custom title in HTML title:
```
Comment (austinchiang) [inline docs/dxp/latest/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md]: Custom value in HTML title:

(keep parallel language as above)

File: docs/dxp/latest/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md [hunk 3]
```diff
@@ -17,15 +17,35 @@ Search engine optimization (SEO) refers to the methods used to improve your Page
...
+![image](https://user-images.githubusercontent.com/67901/137364659-2f22e395-5a6f-46f1-a847-e424706852b3.png)
+
+For example: `${title} - ${authorName}` produces something like.
+
+```
+<title>New awesome feature - Jane Doe </title>
+```
+
+```note::
+   Take into account that we do not provide translation to the text outside mapped fields, we provide translation for mapped files when is possible.
```
Comment (austinchiang) [inline docs/dxp/latest/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md]: Take into account that we do not provide translations to the text outside mapped fields. We provide translations for mapped files when possible.

File: docs/dxp/latest/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md [hunk 4]
```diff
@@ -17,15 +17,35 @@ Search engine optimization (SEO) refers to the methods used to improve your Page
...
+```
 
-![The HTML Title field defines a Display Page's title meta tag](./configuring-seo-and-open-graph/images/02.png)
 
 ### Description
 
-The *Description* field defines a Display Page's description `<meta>` tag. This description is used by search engines to rank your Page and appears in search engine results as a preview of your Page. By default, Display Page templates map this field to *Description*, but you can select any other Text field. The recommended length for a Page's description is under 155 characters.
+The *Description* field defines a Display Page's description `<meta>` tag. This description is used by search engines to rank your Page and appears in search engine results as a preview of your Page. By default, Display Page templates map this field to *Description*, but you can build your description with the mapped fields that the selector gives you.
```
Comment (austinchiang) [inline docs/dxp/latest/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md]: See Slack discussion

File: docs/dxp/latest/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md [hunk 5]
```diff
@@ -49,13 +69,14 @@ Determine whether to include a Display Page in your `sitemap.xml` file, as well
 
 ### Title
 
-The *Title* field defines a Display Page's `og:title` property, which defines the title displayed for your content in rich previews. By default, this field is mapped to *Title*, like the HTML title field for SEO. While you can select any other Text field, it's best practice to update SEO and Open Graph titles together.
+The *Title* field defines a Display Page's `og:title` property, which defines the title displayed for your content in rich previews. By default, this field is mapped to `${title}`, like the HTML title field for SEO. While you can build your title with the mapped fields that the selector gives you, it's best practice to update SEO and Open Graph titles together.
```
Comment (austinchiang) [inline docs/dxp/latest/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md]: it is best to update the SEO and Open Graph titles together.

or

it is best practice to update the SEO and Open Graph titles together.

File: docs/dxp/latest/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md [hunk 6]
```diff
@@ -49,13 +69,14 @@ Determine whether to include a Display Page in your `sitemap.xml` file, as well
...
+
+![image](https://user-images.githubusercontent.com/67901/137367173-6041c671-509f-437a-a245-84d5cf5604e8.png)
 
-![The Title field defines a Display Page's og:title property.](./configuring-seo-and-open-graph/images/06.png)
 
 ### Description
 
-The *Description* field defines a Display Page's `og:description` property, which determines the description displayed for your content in rich previews. By default, Display Page templates map this field to *Description*, like the description field for SEO. While you can select any other Text field, it's best practice to update SEO and Open Graph descriptions together.
+The *Description* field defines a Display Page's `og:description` property, which determines the description displayed for your content in rich previews. By default, Display Page templates map this field to *Description*, like the description field for SEO. While you can build your desciption with the mapped fields that the selector gives you, it's best practice to update SEO and Open Graph descriptions together.
```
Comment (austinchiang) [inline docs/dxp/latest/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md]: property[sic] which determines the description...

Comment (austinchiang) [reply docs/dxp/latest/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md]: just like the description field for SEO

Comment (austinchiang) [reply docs/dxp/latest/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md]: it is best to update the SEO and Open Graph titles together.

or

it is best practice to update the SEO and Open Graph titles together.

**Comment by boton** [comment]
See JamesAGarcia#49.

**Pull request closed without merging**